### ISSUE TITLE
Functions window: stack size

### ISSUE THREAD

**Issue by mumbel** [NationalSecurityAgency/ghidra#6330]
**Is your feature request related to a problem? Please describe.**
Could the functions window have a column for stack frame size?

**Describe the solution you'd like**
New optional column that lists the function's stack size.  Is this always know or easy to access with a quick lookup that this window would need (Would an undefined value be common until maybe prototype committed?)?

**Describe alternatives you've considered**
Not aware of ways to quickly access this

**Additional context**

**Comment by ghidra1** [comment]
Is this specifically StackFrame.getLocalSize() that you are hoping to see?  This is computed based upon the define local variables that utilize the stack.

**Comment by mumbel** [comment]
Yeah, sorry if that wasn't clear.  Just wondering if that value could be in a column in the Functions window.  Like if viewing the function stack editor, the 'Local Size' or 'Frame Size'.  My interest would be sorting by amount of local stack variables.

But I understand if that would be too expensive to compute in that table
